tests/unit/test_utils.py

```diff
@@ -409,3 +409,20 @@ def test_can_escape_recursive_shapes(self):
         }
         self.walker.walk(self.get_shape_model('Recursive'), self.visitor)
         self.assert_visited_shapes(['Recursive'])
+
+
+class TestStreamingBlob(unittest.TestCase):
+    def setUp(self):
+        self.blob = mock.Mock()
+        self.blob.type_name = 'blob'
+        self.blob.serialization = {}
+        self.streaming_blob = mock.Mock()
+        self.streaming_blob.type_name = 'blob'
+        self.streaming_blob.serialization = {'streaming': True}
+        self.non_blob = mock.Mock()
+        self.non_blob.type_name = 'string'
+    
+    def test_is_streaming_blob_type(self):
+        self.assertEqual(is_streaming_blob_type(self.blob), False)
+        self.assertEqual(is_streaming_blob_type(self.streaming_blob), True)
+        self.assertEqual(is_streaming_blob_type(self.non_blob), False)
```

Comment:
For these test assertions, let's break them out into separate test methods. The reason for doing it in separate methods is that if any of the assertions fail, any assertions made past that point of failure will not be run and you will not be aware if an assertion fails until you fix the failing assertion. By spitting it into multiple test cases, you get a better representation of all assertions that are failing.